pdc_client/plugins/compose_full_import.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+
+from __future__ import print_function
+
+import json
+import sys
+import os
+
+from pdc_client.plugin_helpers import PDCClientPlugin, add_create_update_args, extract_arguments
+
+
+class ComposeFullImportPlugin(PDCClientPlugin):
+    def register(self):
+        self.set_command('compose-full-import')
+        create_parser = self.add_action('create', help='create new compose',
```

Comment:
This is compose full import. Actually in pdc's api create compose is a side effect for compose-full-import, so is it proper to mention 'Import RPMs, images and set compose tree location.' as well